lasy/profiles/transverse/flattened_gaussian_profile.py

```diff
@@ -9,92 +11,126 @@ class FlattenedGaussianTransverseProfile(TransverseProfile):
+    The implementation of this class is based on that from `FBPIC`
     <https://github.com/fbpic/fbpic/blob/dev/fbpic/lpa_utils/laser/transverse_laser_profiles.py>.
 
-    **In the focal plane** (:math:`z=z_f`), the profile translates to a
+    **In the focal plane** (:math:`z=z_f`), or in the far field, the profile translates to a
     laser with a transverse electric field:
 
     .. math::
 
         E(x,y,z=zf) \propto
-        \exp\left(-\frac{r^2}{(N+1)w0^2}\right)
-        \sum_{n=0}^N c'_n L^0_n\left(\frac{2\,r^2}{(N+1)w0^2}\right)
+        \exp\left(-\frac{r^2}{(N+1)w_0^2}\right)
+        \sum_{n=0}^N c'_n L^0_n\left(\frac{2\,r^2}{(N+1)w_0^2}\right)
+
+
+    with Laguerre polynomials :math:`L^0_n` and
 
-        \mathrm{with} Laguerre polynomials :math:`L^0_n` and
-        \qquad c'_n = \sum_{m=n}^{N}\frac{1}{2^m}\binom{m}{n}
+    .. math::
+
+        \qquad c'_n=\sum_{m=n}^{N}\frac{1}{2^m}\binom{m}{n}
 
-    - For :math:`N=0`, this is a Gaussian profile: :math:`E\propto\exp\left(-\frac{r^2}{w0^2}\right)`.
+    - For :math:`N=0`, this is a Gaussian profile: :math:`E\propto\exp\left(-\frac{r^2}{w_0^2}\right)`.
 
-    - For :math:`N\rightarrow\infty`, this is a Jinc profile: :math:`E\propto \frac{J_1(r/w0)}{r/w0}`.
+    - For :math:`N\rightarrow\infty`, this is a Jinc profile: :math:`E\propto\frac{J_1(r/w_0)}{r/w_0}`.
 
-    The equivalent expression **far from focus** is
+    The equivalent expression for the collimated beam in the near field which produces this focus is
+    given by:
 
     .. math::
 
         E(x,y,z=\infty) \propto
         \exp\left(-\frac{(N+1)r^2}{w(z)^2}\right)
         \sum_{n=0}^N \frac{1}{n!}\left(\frac{(N+1)\,r^2}{w(z)^2}\right)^n
 
-        \mathrm{with} \qquad w(z) = \frac{\lambda_0}{\pi w0}|z-z_{foc}|
+    with
 
-    - For :math:`N=0`, this is a Gaussian profile: :math:`E\propto\exp\left(-\frac{r^2}{w_(z)^2}\right)`.
+    .. math::
 
-    - For :math:`N\rightarrow\infty`, this is a flat profile: :math:`E\propto \Theta(w(z)-r)`.
+        w(z) = \frac{\lambda_0}{\pi w_0}|z-z_{foc}|
+
+    - Note that a beam defined using the near field definition would be
+      equivalent to a beam defined with the corresponding parameters in
+      the far field, but without the parabolic phase arising from being
+      defined far from the focus.
+
+    - For :math:`N=0`, this is a Gaussian profile: :math:`E\propto\exp\left(-\frac{r^2}{w(z)^2}\right)`.
+    - For :math:`N\rightarrow\infty`, this is a flat profile: :math:`E\propto\Theta(w(z)-r)`.
 
     Parameters
     ----------
-    w0 : float (in meter)
-        The waist of the laser pulse,
-        i.e. :math:`w_{0}` in the above formula.
-    N: int
+    field_type : string
+        Options: 'nearfield', when the beam is defined far from focus and
+        has been collimated, or 'farfield', when the beam is in the vicinity
+        of or has been directly propagated from the focus. In this case there
+        can be a large defocus in the spatial phase.
```

Comment:
I find this slightly misleading, just tried to help clarify, let me know what you think.
```suggestion
        Options: 'nearfield', when the beam is defined far from focus (e.g., right before the focusing optics), or 'farfield', when the beam is in the vicinity of the focus.
```